project/build.scala

```diff
@@ -284,6 +286,19 @@ object build {
     }
   )
 
+  private[this] val scala213_pre_cross_setting = {
+    // sbt wants `scala-2.13.0-M1`, `scala-2.13.0-M2`, ... (sbt/sbt#2819)
+    // @fommil tells me we could use sbt-sensible for this
+    unmanagedSourceDirectories in Compile ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2L, minor)) =>
+          Some((baseDirectory in Compile).value.getParentFile / s"src/main/scala-2.$minor")
+        case _               =>
+          None
+      }
+    }
+  }
```

Comment:
@fommil wdyt

Reply:
I love the way it's a `Long`.

Implying that the madness will not stop with Scala 3.

Reply:
I expect Scala to be the perfect language by version 34359738368.103079215116.2